sys/shell/shell.c

```diff
@@ -225,7 +225,7 @@ static int readline(shell_t *shell, char *buf, size_t size)
         if (c == '\r' || c == '\n') {
             if (line_buf_ptr == buf) {
                 /* The line is empty. */
-                continue;
+                return 1;
             }
 
             *line_buf_ptr = '\0';
```

Comment:
If you press enter twice I expect each new prompt to be displayed on a new
line, but it isn't.
Expected:
`> `
`> `

Actual:
`> > `